pulsar-common/src/test/java/org/apache/pulsar/common/policies/data/ClusterDataTest.java

```diff
@@ -109,4 +110,80 @@ public void simple() {
                         .build().hashCode());
 
     }
+
+    @Test
+    public void testCheckProperties() {
```

Comment:
Maybe we can use a data provider here.

Reply:
Because this test logic includes exceptions and normal cases, if we convert this to the data provider, we need to add more `if-else` to predicate the result. It will introduce more complexities.